### PR TITLE
adding parameters that disable OAB and TSB

### PR DESCRIPTION
**Pull request by kalexand-rh**
Fixes https://github.com/openshift/openshift-docs/issues/9889

Per @sdodson in IRC

@jiajliu, will you PTAL?

### CODE REVIEW

**Comment by sdodson** [comment]
lgtm, only thing I can think of is that these three components are linked together, it may be worth considering placing the ASB and TSB as subheadings of the Service Catalog heading here.

**Comment by kalexand-rh** [comment]
@jianlinliu PTAL

**Comment by jianlinliu** [comment]
LGTM, the only suggestion is that replace OAB with ASB.

**Comment by kalexand-rh** [comment]
@jianlinliu, OAB is used far more frequently in the docs than ASB. If we need to update all the references, please file a bug.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -1734,17 +1734,30 @@ inventory file:
 ----
 openshift_enable_service_catalog=false
 ----
+
+If you disable the service catalog, the OpenShift Ansible broker and template
+service broker are not installed.
 ====
 
 When the service catalog is enabled, the OpenShift Ansible broker and template service broker are both enabled
 as well; see xref:configuring-openshift-ansible-broker[Configuring the OpenShift Ansible Broker] and xref:configuring-template-service-broker[Configuring the Template Service Broker] for more information.
```
Comment (bfallonf) [inline install/configuring_inventory_file.adoc]: In the least, I think what we're adding can be combined with the sentence above the Ansible variable. So:

> [NOTE]
> ====
> To not install the OpenShift Ansible broker and the template service broker, set the service catalog variable in the Ansible inventory file at installation time to `false`:
> 
> ----
> openshift_enable_service_catalog=false
> ----
> ====

But, this section confuses me. It's called "Configuring the service catalog" and the only point is that it's enabled by default, and then there's a thing on how to disable it, but it's in a note box? If anything, I think "Enabling or Disabling the service catalog at installation" might be a better title, and getting rid of the note box.

Sorry, I realise this was like this when you got here...

Comment (bfallonf) [reply install/configuring_inventory_file.adoc]: I think similar comments can be made about the other two sections you've touched below. Up to you how deep you want to go, though.

**Comment by bfallonf** [comment]
@kalexand-rh I've made some comments. Feel free to implement them or throw them out the window. Otherwise, LGTM :door:

**Comment by kalexand-rh** [comment]
Manual CPs for 3.9 and 3.7 because the change needs to go into a different file.

**Pull request merged**